### PR TITLE
chrootenv: strip the binary

### PR DESCRIPTION
**Pull request by pbogdan**
###### Motivation for this change

Get rid of references to the full compiler package from the runtime closure.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

Not sure how to test this directly as `chrootenv` is not publicly exposed AFAIK. Tested indirectly as part of `steam`.

/cc maintainer @yegortimoshenko

### CODE REVIEW

**Comment by pbogdan** [comment]
I'm confused by the rebuild amount - is Hydra now building unfree software?

File: pkgs/build-support/build-fhs-userenv/chrootenv/default.nix [hunk 1]
```diff
@@ -1,19 +1,15 @@
...
 
-  buildCommand = ''
-    cc ${./chrootenv.c} $(pkg-config --cflags --libs glib-2.0) -o $out
-    ${stdenv.cc.bintools.bintools}/bin/strip $out
-  '';
-
   meta = with stdenv.lib; {
     description = "Setup mount/user namespace for FHS emulation";
-    license = licenses.free;
+    license = licenses.mit;
```
Comment (yegortimoshenko) [inline pkgs/build-support/build-fhs-userenv/chrootenv/default.nix]: That is, Nixpkgs license.

**Comment by yegortimoshenko** [comment]
Thanks for the catch! This really shouldn't override default phases. I've pushed a commit that uses Meson to build the project. Meaning, no custom phases are necessary at all. What do you think?

cc @abbradar, @jtojnar

**Review by jtojnar — APPROVED** [review]

**Comment by pbogdan** [comment]
LGTM, thanks!

**Pull request merged**